packages/components/src/components/contract-card/contract-card-items/contract-card-body.jsx

```diff
@@ -146,6 +148,7 @@ const ContractCardBody = ({
     contract_update,
     currency,
     current_focus,
+    error_message_alignment,
```

Comment:
Isn't it better to add the type of this to `propTypes`?

Reply:
Yes, will do 👍